libbeat/cmd/instance/beat.go

```diff
@@ -106,9 +110,6 @@ type beatConfig struct {
 	Template   *common.Config `config:"setup.template"`
```

Comment:
I think we should get rid of this config here, to avoid any conflicts with what is going on in the `idxmgmt.Supporter`

Reply:
done